### ISSUE TITLE
Understanding Content   on Hover or Focus

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/wcag#348]
_From @awkawk on March 2, 2018 15:20_

<p>The <a   href="https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus"   target="_blank">1.4.13 Content on Hover or Focus (AA)   SC</a> has draft Understanding content that can be viewed at <a   href="https://rawgit.com/w3c/wcag21/content-on-hover-or-focus/understanding/21/content-on-hover-or-focus.html"   target="_blank">Understanding Content on Hover or   Focus</a>.</p><p>Please review the Understanding document   and submit a comment on this issue for any changes that are needed to the   content.</p>

_Copied from original issue: w3c/wcag21#784_

**Comment by michael-n-cooper** [comment]
_From @alastc on April 18, 2018 16:50_

@guyhickling noted that the understanding doc doesn't match the SC text anymore in issue #807

**Comment by michael-n-cooper** [comment]
_From @mbgower on April 19, 2018 18:4_

Created minor changes to contain visible trigger language to align with SC wording . https://github.com/w3c/wcag21/pull/869
@steverep you may want to review.

**Comment by michael-n-cooper** [comment]
_From @mbgower on April 19, 2018 22:21_

Review complete; changes submitted via git and pull request 869 completed.
I reviewed the last public comments closed against this SC, and confirmed that some information on the user agent exception is provided (in the Additional Notes subsection). That info could be potentially more prominent or elaborated upon; however, in combination with the wording of the SC and the two Notes that appear under the SC wording, I think the context is pretty clear, even if the rationale for why the exception exists isn't explained.

**Comment by michael-n-cooper** [comment]
_From @lseeman on April 22, 2018 17:16_

under Hoverable i would add that the content should stay visable while the pointer in on any  direct paths between the triger and the new content . Otherwise the content disapears

**Comment by michael-n-cooper** [comment]
_From @lseeman on April 22, 2018 17:17_

Likewise 
The user removes hover or focus from the trigger and the additional content, consistent with the typical user experience
should be
The user removes hover or focus from the trigger and the additional content and any direct path between them, consistent with the typical user experience

**Comment by michael-n-cooper** [comment]
_From @alastc on April 22, 2018 21:56_

> content should stay visible while the pointer in on any direct paths between the trigger and the new content . Otherwise the content disappears

Some of the better hoverable menus use a time-out to control when the additional content disappears which helps with cutting corners. For example, if you go to [John Lewis](https://www.johnlewis.com/) and hover over 'furniture and lights', then move the mouse to 'lighting' on the right, it doesn't matter if you clip 'electrical'. Saying 'direct paths' doesn't work for that.

I don't think that is a reasonable requirement to add, especially in the understanding document.

**Comment by michael-n-cooper** [comment]
_From @alastc on April 24, 2018 8:11_

From #808 we agreed that the third bullet in 'Additional notes' should be:
> Content which can be triggered via pointer hover must also be accessible via the keyboard. Refer to Success Criterion 2.1.1, Keyboard.

Also, consider adding a note below that to the effect that it's bad usability to display things on focus and on hover (rather than after an explicit action by the user).

**Comment by michael-n-cooper** [comment]
_From @mbgower on May 4, 2018 21:20_

For clarifcation, I am closing PR https://github.com/w3c/wcag21/pull/869. This was accidentally made as a pull to Master. My changes were already entered into the correct working branch for the understanding doc as 2 commits (https://github.com/w3c/wcag21/pull/869/commits/3526b55e2827ed8ccb843fd76148271ca052ecde, https://github.com/w3c/wcag21/pull/869/commits/deb55c174971c6838bdfecf56f691fd7544e6450)

However, I did want to capture here the comments @awkawk put into that PR here, before I closed it:
> I'd like to have more generic examples, that is, not using the WCAG 2.1 GitHub repo as the source of the example. This example (#1) would also pass since it is implemented by the browser rather than the author.
I also think that we need to show an example of the intersection of tooltip hovering and screen magnification if we can.

**Comment by guyhickling** [comment]
I support Michael in his comment above a couple of hours ago:

> Also, consider adding a note below to the effect that it's bad usability to display things on focus

Anything that can be said to discourage this practice is good, as displaying on focus means keyboard users have to look at it without having any choice, and screen reader users have to listen to it every time they land on it.

**Comment by fstrr** [comment]
This issue looks to be inactive, so I'm closing it. Please re-open if necessary.
